godtools/download/action.yaml

```
@@ -0,0 +1,26 @@
+name: 'Piwik PRO godtools download'
+description: 'Downloads content from artifactory'
+inputs:
+  godtools-config:
+    required: true
+    description: godtools encrypted configuration
+  godtools-key:
+    required: true
+    description: godtools encryption key
+  data:
+    required: true
+    description: What to download
```

`data` feels too generic for me. What about `artifacts`?
```suggestion
  artifacts:
    required: true
    description: Type of artifacts to download (events etc.)
```